Classes/SchedulerTasks/MailNotifier.php

```diff
@@ -183,7 +182,8 @@ private function sendRemindersToOrganizers(\Tx_Seminars_OldModel_Event $event, $
         foreach ($event->getOrganizerBag() as $organizer) {
             /** @var \Tx_Oelib_Mail $eMail */
             $eMail = GeneralUtility::makeInstance(\Tx_Oelib_Mail::class);
-            $eMail->setSender($sender);
+            $eMail->setSender($event->getEmailSender());
```

Comment:
We should keep the local variable `$event` to avoid multiple calls to `$event->getEmailSender()` in the loop.